gnosis/safe-react
# Chore: update remote config mocks on commit

## What it solves
Resolves #3161

## How this PR fixes it
Adds a new script and npm command to re-fetch the config and store in a mock file.

## How to test it
* Modify the mock file to make it "stale"
* Try making a commit
* It should update the mock
